ocean-client/src/utils/store_config.ts

```diff
@@ -93,9 +104,13 @@ class ConfigFile {
   seedfile: string = 'V:/store/vault-maxi-seed.txt'
   minCollateralRatio: number = 200
   maxCollateralRatio: number = 250
-  LMToken: string = 'GLD'
-  LMPair: string | undefined
+  LMToken: string | undefined
+  LMPair: string = 'GLD-DUSD'
   mainCollateralAsset: string = 'DFI'
   reinvestThreshold: number | undefined = 0
+  reinvestPattern: string = 'DFI'
```

Comment:
should not need a default. internally, if its undefined its set to the mainCollateralAsset

Reply:
I will remove the default value